static/docs/commands-reference/get.md

```diff
@@ -42,7 +42,8 @@ created in the current working directory, with its original file name.
   isn't used) is the current working directory (`.`) and original file name.
 
 - `--rev` - specific Git revision of the DVC repository to import the data from.
-  `HEAD` by default.
+  [`HEAD`](https://git-scm.com/book/en/v2/Git-Internals-Git-References#ref_the_ref)
+  is used by default when this option is not specified.
```

Comment:
> Moved from https://github.com/iterative/dvc.org/pull/591#discussion_r319361679

Maybe it's more correct to say `master`? Or "tip of the default branch", @efiop?

Reply:
Ping @efiop

Reply:
OK well, I went for "tip of the default branch" in 5fe82e7.